Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.android.js

```diff
@@ -216,6 +217,7 @@ var styles = StyleSheet.create({
     position: 'absolute',
     top: 0,
     bottom: 0,
+    backgroundColor: 'white',
```

Comment:
Yes the shadow doesn't draw if the view doesn't have a backgroundColor. I don't think it will ever be visible unless someone doesn't set any background to their drawer view in their app.